odd_collector/adapters/postgresql/adapter.py

```diff
@@ -26,17 +27,31 @@ def get_data_entity_list(self) -> DataEntityList:
         with PostgreSQLRepository(
             ConnectionParams.from_config(self.config), self.config.schemas_filter
         ) as repo:
+            table_entities: list[DataEntity] = []
+            schema_entities: list[DataEntity] = []
             tables = repo.get_tables()
-
-            table_entities = map_tables(generator=self.generator, tables=tables)
+            schemas = repo.get_schemas()
+            self.generator.set_oddrn_paths(**{"databases": self.config.database})
+
+            tables_by_schema = {}
```

Comment:
You can use `groupby` or `defaultdict` instead. it will simplify your code.

Reply:
Done